adapters/Vungle/VungleAdapter/GADMAdapterVungleRouter.m

```diff
@@ -204,24 +212,46 @@ - (nullable NSError *)loadAd:(nonnull NSString *)placement
     [delegate adAvailable];
   } else {
     NSError *loadError;
-    if (![sdk loadPlacementWithID:placement error:&loadError]) {
-      if (loadError) {
-        return loadError;
-      }
+    GADMAdapterVungleAdType adType = [delegate adapterAdType];
+    if (adType != GADMAdapterVungleAdTypeBanner && adType != GADMAdapterVungleAdTypeShortBanner && adType != GADMAdapterVungleAdTypeLeaderboardBanner) {
```

Comment:
So Vungle MREC ads are requested differently than the rest of banners? Maybe in that case we need 2 enum values, 1 for MREC and only one other for "banner"?

Reply:
@joshuajplagonera Hi joshuajplagonera, yes, Vungle MREC ads and Vungle Banner ads are using different APIs to load ads. Same question as above, all of these Banner enums are needed. When loading Vungle Banner ads, we also need to pass the Banner size type (Banner[320 * 50], short Banner[300 * 50] or Leaderboard Banner[728*90]) to Vungle SDK, so we need different Banner enum types to distinguish Banner ads of different sizes. Thanks a lot. 🙂